Makefile

```diff
@@ -16,6 +16,8 @@ OBJCOPY := $(DEVKITARM)/bin/arm-none-eabi-objcopy
 LIBGCC := tools/agbcc/lib/libgcc.a
 LIBC   := tools/agbcc/lib/libc.a
 
+LIBC := tools/agbcc/lib/libc.a
```

Comment:
This line is no longer necessary. (It's' repeated two lines above)

Reply:
merge must have added this. whoops.